dist-apidoc target give errors

Running `make dist-apidoc` produces the following errors:
node node_modules/.bin/jsdoc -c build/jsdoc/api/conf.json -d dist/apidoc
ERROR: Unable to create a Tag object for source file /home/gberaudo/dev/cesiumv1.1/externs/olcsx.js with title "type" and body " {(function(!ol.Map, !Cesium.Scene): Array.<olcs.AbstractSynchronizer>)|undefined}
": Invalid type expression "(function(!ol.Map, !Cesium.Scene): Array.<olcs.AbstractSynchronizer>)|undefined": Expected "!", "=" or "?" but "|" found.
ERROR: Unable to create a Tag object for source file /home/gberaudo/dev/cesiumv1.1/externs/olcsx.js with title "type" and body " {(function(!ol.Map, !Cesium.Scene): Array.<olcs.AbstractSynchronizer>)|undefined}
": Invalid type expression "(function(!ol.Map, !Cesium.Scene): Array.<olcs.AbstractSynchronizer>)|undefined": Expected "!", "=" or "?" but "|" found.
Makefile:48: recipe for target 'dist-apidoc' failed
make: *** [dist-apidoc] Error 1

A dist/apidoc directory is created though.

--- a/externs/olcsx.js
+++ b/externs/olcsx.js
@@ -10,7 +10,7 @@ var olcsx;
  * @typedef {{
  *   map: (!ol.Map),
  *   target: (Element|string|undefined),
- *   createSynchronizers: ((function(!ol.Map, !Cesium.Scene): Array.<olcs.AbstractSynchronizer>)|undefined)
+ *   createSynchronizers: (undefined|function(!ol.Map, !Cesium.Scene): Array.<olcs.AbstractSynchronizer>)
  * }}
  * @api
  */
@@ -38,7 +38,7 @@ olcsx.OLCesiumOptions.prototype.target;
  * constructor to create custom synchronizers. Receives an `ol.Map` and a
  * `Cesium.Scene` as arguments, and needs to return an array of
  * {@link olcs.AbstractSynchronizer}.
- * @type {(function(!ol.Map, !Cesium.Scene): Array.<olcs.AbstractSynchronizer>)|undefined}
+ * @type {undefined|function(!ol.Map, !Cesium.Scene): Array.<olcs.AbstractSynchronizer>}
  * @api
  */
 olcsx.OLCesiumOptions.prototype.createSynchronizers;

